Servers/UI/OJS.Servers.Ui/ClientApp/src/layout/content/PageContent.tsx

```diff
@@ -31,27 +32,27 @@ const routes = [
     },
     {
         path: '/',
-        Element: HomePage,
+        Element: withTitle(HomePage, 'SoftUni Judge'),
     },
     {
         path: '/profile',
-        Element: ProfilePage,
+        Element: withTitle(ProfilePage, 'My Profile'),
```

Comment:
Better to add a new nullable property `title` along with `path` and `Element`, i.e:
```
{
    path: '/profile',
    Element: ProfilePage,
    title: 'My profile',
}
```

Then all titles to all pages in `Routes`